### PR TITLE
Update spider name to chi school CAC from CPS CAC

### PR DESCRIPTION
**Pull request by o-stovicek**
From issue #358
cps_community_action_council to chi_school_community_action_council

### CODE REVIEW

**Comment by r-wei** [comment]
Looks good to me! @pjsier , is there anything else that needs to be updated for deployment?

**Comment by pjsier** [comment]
This is a small thing, but can we capitalize the first letter in the class name so that it's `Chi_school_community_action_council` for consistency? This would only apply to the actual class, so the `name` attribute would stay the same. Otherwise it looks great!

**Comment by r-wei** [comment]
Hey @o-stovicek , can you rename test_chi_school_community_action_council to test_chi_school_community_action_council.py? Our testing framework needs all of our test files to end in `.py` in order to run.

File: tests/test_chi_school_community_action_council [hunk 1]
```diff
@@ -2,10 +2,10 @@
 import pytest
 
 from tests.utils import file_response
-from city_scrapers.spiders.cps_community_action_council import Cps_community_action_councilSpider
+from city_scrapers.spiders.chi_school_community_action_council import chi_school_community_action_councilSpider
```
Comment (pjsier) [inline tests/test_chi_school_community_action_council]: Small thing, but it needs to be the same renamed class name with a capital C here, `Chi_school_community_action_councilSpider`

File: tests/test_chi_school_community_action_council [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 
-test_response = file_response('files/cps_community_action_council_CAC.html', url='http://cps.edu/FACE/Pages/CAC.aspx')
-spider = Cps_community_action_councilSpider()
+test_response = file_response('files/chi_school_community_action_council_CAC.html', url='http://cps.edu/FACE/Pages/CAC.aspx')
+spider = chi_school_community_action_councilSpider()
```
Comment (pjsier) [inline tests/test_chi_school_community_action_council]: Same capital C here

**Comment by o-stovicek** [comment]
Sorry about missing those capital letters and the .py! Just starting to learn about coding and did not catch when I messed those things up. I think I altered everything you mentioned, but now the checks aren't passing, so I'll try to look at what's causing that

**Comment by o-stovicek** [comment]
Okay, checks passing now! Would appreciate someone taking a look with all the small changes; apologies for doing this somewhat messily

**Comment by r-wei** [comment]
No worries @o-stovicek :) looks good to me

**Pull request merged**